src/cdk-experimental/scrolling/virtual-scroll-viewport.ts

```diff
@@ -275,7 +272,37 @@ export class CdkVirtualScrollViewport implements DoCheck, OnInit, OnDestroy {
     }
   }
 
-  /** Sets the scroll offset on the viewport. */
+  /**  Scrolls to the offset on the viewport. */
+  scrollToOffset(offset: number, options = { smooth: false, lazy: false }) {
+    const viewportElement = this.elementRef.nativeElement;
+    const top = this.orientation === 'vertical' ? offset : 0;
+    const left = this.orientation === 'horizontal' ? offset : 0;
+
+    let shouldScroll = true;
+    if (options.lazy) {
```

Comment:
I'm not sure if this logic is quite right because the rendered content may be larger than the viewport.

Reply:
Your right. I changed it to measure the viewport size rather than content.

Reply:
I'm wondering if lazy scrolling is a common enough use case to be something we want to include in the API. It's pretty easy to implement on top if we just provide the basic functionality